test/Thermal/motor.jl

```diff
@@ -0,0 +1,56 @@
+using ModelingToolkit, OrdinaryDiffEqDefault, Test
```

Comment:
These new files weren't added to runtests? Please fix today or I need to revert.